src/tenant/tests/test_admin.py

```diff
@@ -235,7 +250,7 @@ def test_search_on_custom_fields(self):
             reverse("admin:{}_{}_changelist".format("tenant", "tenant")) + "?q=doe.com"
         )
         # confirm the search returned one object (by email address)
-        self.assertContains(response, "1 result")
+        self.assertContains(response, "2 result")
```

Comment:
Code comment says return one object but you are returning two... 

`# confirm the search returned one object (by email address)`

Reply:
correct, it was my mistake. it should say "two objects", since after this patch "change list" view shows both verified and unverified email

Reply:
should be corrected in a latest commit